Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -10,11 +10,25 @@ attribute vec4 eyeOffset;                                  // eye offset in mete
 attribute vec4 scaleByDistance;                            // near, nearScale, far, farScale
 attribute vec4 pixelOffsetScaleByDistance;                 // near, nearScale, far, farScale
 attribute vec3 distanceDisplayConditionAndDisableDepth;    // near, far, disableDepthTestDistance
+#ifdef CLAMP_TO_GROUND
+attribute vec4 textureOffset;                              // the min and max x and y values for the texture coordinates
+attribute vec2 dimensions;
+#endif
 #ifdef VECTOR_TILE
 attribute float a_batchId;
 #endif
 
 varying vec2 v_textureCoordinates;
+#ifdef CLAMP_TO_GROUND
```

Comment:
This is going to use a huge amount of memory bandwidth.

Looks easy to pack some varyings together.  Also perhaps some can be derived from others or more should be computed in the fragment shader if they are trivial?

Reply:
Also, use more precise names - if possible - than `1`, `2`, and `3` for texture coordinates.